package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Wed Dec 15 18:48:46 UTC 2021 - David Diaz <dgonzalez@suse.com>
+
+- Add support for improving the handling if the user decides to
```

Comment:
is it really just support or it do real fix to respect abort from user?

Reply:
I would change the description. All this PR is about raising exception when probing the system.